themes/kiali/layouts/partials/home/about-section.html

```diff
@@ -1,19 +1,18 @@
   <section id="about">
-    <h1>About Kiali</h1>
+    <h3>About Kiali</h3>
 
     <p>
-    Kiali is a management console for an Istio-based service mesh. It provides
-    dashboards, observability, and lets you operate your mesh with robust
-    configuration and validation capabilities. It shows the structure of your
-    service mesh by inferring traffic topology and displays the health of your
-    mesh. Kiali provides detailed metrics, powerful validation, Grafana access,
-    and strong integration for distributed tracing with Jaeger.
+    Kiali is a management console for Istio service mesh. It provides robust
```

Comment:
Style comments are pretty subjective from all angles.
As it happens with presentations with long text, this is a little bit hard to read:
![image](https://user-images.githubusercontent.com/1662329/132227625-988dcd31-a691-419e-856d-7065511105a1.png)

I don't think it has to be comprehensive, just to invite to jump to the next area "Features" and from that to the documentation.

To me it's hard to read 5 lines of text with small font in a first page.

Reply:
For me this is OK, leaving as-is.  I had already updated the text to hopefully be easier to read and more current.  With the other changes it's quite easy to go past this to the Features section, and for some people (like me) I like reading a short blurb.